pkg/cmd/build/build.go

```diff
@@ -43,6 +43,8 @@ import (
 const (
 	warningDockerfilePath   string = "Build '%s': Dockerfile '%s' is not in a relative path to context '%s'"
 	doubleDockerfileWarning string = "Build '%s': Two Dockerfiles discovered in both the root and context path, defaulting to '%s/%s'"
+	maxArgFormatParts              = 2
+	minArgFormatParts              = 1
```

Comment:
I am not sure if it is correct to have this value as a constant. I think you should reduce its scope to the function where it is used but I would not reuse its value under a constant, I see it quite error prone as each function may require strings.SplitN to split it into N.

Reply:
There are quite a few usages of these 2 vars. Being 1 and 2 I think it's safe to use them as `const`. How do you see this be error prone?

Reply:
What I mean is that using this as const for strings.splitN because they share the same value for me is like reusing both values because they happen to share value but not because in the places where they are used they have to have always the same value as to use the same constant in both cases

Reply:
now that we have added strings.splitn in the list,  I think that using the magic number directly in strings.splitn makes more sense than reusing it through a constant.